lib/vmdb/config.rb

```diff
@@ -259,7 +259,7 @@ def set_worker_setting!(klass, setting, value)
       config.store_path(keys, value)
     end
 
-    def save
+    def save(save_db_file = true)
```

Comment:
Good point. :D No big deal either way then +1

On Mon, Feb 15, 2016 at 4:41 PM, Joe Rafaniello <notifications@github.com>
wrote:

> In lib/vmdb/config.rb
> <https://github.com/ManageIQ/manageiq/pull/6675#discussion_r52946111>:
>
> > @@ -259,7 +259,7 @@ def set_worker_setting!(klass, setting, value)
> >        config.store_path(keys, value)
> >      end
> >
> > -    def save
> > +    def save(save_db_file = true)
>
> No way, hash rockets for life [image: :trollface:]
>
> I purposely made it non-obvious because it's really never supposed to be
> used like this except for these tests. I can do keyword arguments if that's
> what everyone thinks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/6675/files#r52946111>.
>



-- 
Chris Arcand
@chrisarcand